javascript/javascript1/week4/homework/code-wars-homework.js

```diff
@@ -0,0 +1,73 @@
+// Exercise 1 - Vowel Count
+
+const testStr = "this is a test string";
+
+function vowelCounter(string) {
+  // Method 1 - regular expression ( I asked AI to generate the regular expression.)
+  const regex = /[aeiouAEIOU]/g; // Regular expression that checks for the given vowels be it lower or upper case.
+  const vowels = string.match(regex);
+
+  //return `The string contains ${vowels.length} vowels.`; // Uncomment to test.
+
+  // Method 2 - for loops
+  const vowelsArray = [];
+  // Simple for loop
+  for (let i = 0; i < string.length; i++) {
+    if (string[i] === "a") {
+      vowelsArray.push(string[i]);
```

Comment:
A succession of `if () ... else if () ...` is fine. However, you can make your code more readable using a [`switch`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch)